src/H5Ppublic.h

```diff
@@ -6445,14 +6445,20 @@ H5_DLL herr_t H5Pset_alloc_time(hid_t plist_id, H5D_alloc_time_t alloc_time);
  *
  * \note Chunk size cannot exceed the size of a fixed-size dataset. For
  *       example, a dataset consisting of a 5x4 fixed-size array cannot be
- *       defined with 10x10 chunks. Chunk maximums:
- *       - The maximum number of elements in a chunk is 2<sup>32</sup>-1 which
- *         is equal to 4,294,967,295. If the number of elements in a chunk is
- *         set via H5Pset_chunk() to a value greater than 2<sup>32</sup>-1,
- *         then H5Pset_chunk() will fail.
- *       - The maximum size for any chunk is 4GB. If a chunk that is larger
- *         than 4GB attempts to be written with H5Dwrite(), then H5Dwrite()
- *         will fail.
+ *       defined with 10x10 chunks.
+ *
+ * \note As of HDF5 version 2.0.0, creation of datasets with arbitrarily large
```

Comment:
Should we avoid the word "arbitrarily" in several places, as the size is still currently limited to 64-bit sizes?